src/Telegram.Bot.Core/Requests/Available Methods/Admin Bot/PromoteChatMemberRequest.cs

```diff
@@ -8,75 +7,66 @@ namespace Telegram.Bot.Requests
     /// <summary>
     /// Promote or demote a user in a supergroup or a channel. The bot must be an administrator in the chat for this to work and must have the appropriate admin rights. Pass False for all boolean parameters to demote a user.
     /// </summary>
-    [JsonObject(MemberSerialization.OptIn, NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
-    public class PromoteChatMemberRequest : RequestBase<bool>
+    public sealed class PromoteChatMemberRequest : ChatIdRequestBase<bool>
```

Comment:
Why is it sealed?

Reply:
Why not? It makes no sence. If someone needs to change the method name, he/she will implement a new class